Tools/cases_generator/generators_common.py

```diff
@@ -13,7 +14,7 @@
 from cwriter import CWriter
 from typing import Callable, TextIO, Iterator, Iterable
 from lexer import Token
-from stack import Storage, StackError
+from stack import Storage, StackError, Stack
```

Comment:
```suggestion
from stack import Storage, StackError
```
`Stack` is not used.